Make dokumentation litlebit more foolproof. (#183)
cakephp/authorization: docs/en/request-authorization-middleware.rst

========

Create a policy for handling the request object. The plugin ships with an

> interface here to implement, create ``src/Policy/RequestPolicy.php``  ::

Reviewer: ```suggestion
interface we can implement. Start by creating **src/Policy/RequestPolicy.php** and add::
```

        }
    }

Map the request class to the policy inside ``Application::getAuthorizationService()``, in **src/Application.php** ::

    use Authorization\AuthorizationService;
    use Authorization\AuthorizationServiceInterface;
    use Authorization\AuthorizationServiceProviderInterface;
    use Authorization\Middleware\AuthorizationMiddleware;
    use Authorization\Middleware\RequestAuthorizationMiddleware;
    use Authorization\Policy\OrmResolver;
    use Psr\Http\Message\ResponseInterface;
    use App\Policy\RequestPolicy;
    use Authorization\Policy\MapResolver;

>     use Cake\Http\ServerRequest;

Reviewer: I can fix this post-merge.
